linters/.markdownlint.json

```diff
@@ -0,0 +1,33 @@
+{
+  "comment": "Enable all rules, and then disable selective ones https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md",
+  "default": true,
+
+  "line-length": false,
+  "ul-start-left": false,
+  "no-multiple-blanks": false,
+  "no-emphasis-as-header": false,
+
+  "comment": "Needed for explicit anchors",
+  "no-inline-html": false,
+
+  "comment": "Rules use '-', some other lists use '+'",
+  "ul-style": false,
+
+  "comment": "LICENCE.md does not conform",
```

Comment:
Great point.

It turns out that `markdownlint-cli` wasn't handling recursive wildcards the way I expected, and I only have shell wildcards at my disposal. I'm going to restrict this PR to just linting style guides for now, rather than all of the `.md` files in the repo, and that will take care of not linting `LICENSE.md`.

I'm curious, though, why does the `LICENSE` file have a `.md` extension in the first place? It doesn't have any markup in it.